cmd/cluster-kube-controller-manager-operator/render/render.go

```diff
@@ -92,9 +104,29 @@ func (r *renderOpts) Complete() error {
 	return nil
 }
 
+type TemplateData struct {
+	genericrenderoptions.ManifestConfig
```

Comment:
better embed `genericrenderoptions.TemplateData` to not loose anything when `genericrenderoptions.TemplateData` is extended in library-go

Reply:
TemplateData includes ManifestConfig already

Reply:
same for FileConfig